src/params.rs

```diff
@@ -83,6 +83,7 @@ pub struct Params {
   pub inputs: Vec<Array>,
   pub outputs: Vec<Array>,
   pub recurrences: Vec<Array>,
+  pub deltaRec: Vec<Array>,
```

Comment:
Looks like you use this once only, does it need to be a vector? Also, Rust notation isn't camelCase, but _ separated: `delta_rec`